src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,10 @@
 
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T> {
     /**
```

Comment:
According to the checklist, don't begin interface or class implementation with an empty line. Remove the empty line at the start of the interface body.